spec/std/crypto/bcrypt_spec.cr

```diff
@@ -47,4 +47,12 @@ describe "Bcrypt" do
     hash = Crypto::Bcrypt.digest("secret", 5)
     Crypto::Bcrypt.verify("Secret", hash).should be_false
   end
-end
\ No newline at end of file
+
+  it "check the speed" do
+    time = Time.now
+    Crypto::Bcrypt.digest("secret")
+    time = Time.now - time
+    (time.to_f < 0.07).should be_true # at least faster than ruby ;)
+  end
```

Comment:
The load can change between that measurement and executing the "real" thing, especially on CI servers. I'm sorry but the idea of benchmark specs is sadly quite inherently flawed.